packages/desktop-client/src/hooks/useLocalPref.ts

```diff
@@ -0,0 +1,9 @@
+import { useSelector } from 'react-redux';
+
+import { type LocalPrefs } from 'loot-core/types/prefs';
+
+export function useLocalPref<K extends keyof LocalPrefs>(
+  prefName: K,
+): LocalPrefs[K] {
+  return useSelector(state => state.prefs.local?.[prefName]);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏my wishlist item (but don't feel pressured to actually implement this) - for these hooks to have both setter & getter and a default value.

```
const [myPref, setMyPref] = useLocalPref('budget.collapsed', true);

// Usage:
console.log('my pref:', myPref);
<Button onClick={(someNewValue) => setMyPref(someNewValue)}>Click me</Button>
```

Reply:
This is a good idea! Implemented this :)